File: src/test/java/chapter/seven/nine/two/one/HrefMatchTest.java
package chapter.seven.nine.two.one;

import org.junit.jupiter.api.Test;

import java.lang.reflect.Array;
import java.util.Arrays;

class HrefMatchTest {

    @Test
    void maintest() {

        System.out.println(Arrays.toString(  MockTest.MockTest("https://www.eduexam.cn/\n", () -> HrefMatch.main(new String[]{}))));
    }
}

File: src/test/kotlin/chapter/seven/nine/two/one/HrefMatchTest.kt
package chapter.seven.nine.two.one

import com.github.masx200.learn_java_kotlin_programming.utils.MockTest
import org.junit.jupiter.api.Test

internal class HrefMatchTest {
    @Test
    fun maintest() {
        println(MockTest.MockTest("https://www.eduexam.cn/\n") { HrefMatch.main(arrayOf()) }
            .contentToString())
    }
}
